modules/ui/src/main/scala/lucuma/ui/primereact/FormInputText.scala

```diff
@@ -62,12 +62,11 @@ object FormInputText {
         onKeyDown = props.onKeyDown,
         modifiers = props.modifiers
       ),
-      props.postAddons.zipWithIndex.toVdomArray { (p, i) =>
```

Comment:
@cquiroz, I assume this was to fix the same error?

Reply:
Yes, it gives a key based on position

Reply:
Right. This fix eliminates the need for a key. I just wanted to make sure you didn't need the key for something.